### PR TITLE
Enable stale action to handle aging issues and pull requests

### PR DESCRIPTION
**Pull request by bunchesofdonald**
This enables the [stale action](https://github.com/actions/stale), but sets it in debug mode so that we can look at the changes it would make if it were fully enabled. This way we can change the settings around till we're happy.

To see the debug output we'll have to set the `ACTIONS_STEP_DEBUG` secret in the repo to `true`.

### CODE REVIEW

**Review by madkinsz — APPROVED** [review]
I do not have permissions to add the secret, cc @abrookins 

I'm basically fine to turn this on without debugging too though.

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          enable-statistics: true
+          stale-issue-message: "This issue is stale because it has been open 30 days with no activity. To keep this issue open remove stale label or comment."
+          close-issue-message: "This issue was closed because it has been stale for 7 days with no activity. If this issue is important or you have more to add feel free to re-open it."
+          stale-pr-message: "This pull request is stale because it has been open 30 days with no activity. To keep this pull request open remove stale label or comment."
+          close-pr-message: "This pull request was closed because it has been stale for 7 days with no activity. If this pull request is important or you have more to add feel free to re-open it."
+          days-before-stale: 30
+          days-before-close: 7
```
Comment (cicdw) [inline .github/workflows/stale.yml:19]: 7 seems tight to me, I'd probably vote to be more generous with 14 but easy enough to change

**Review by cicdw — APPROVED** [review]
I added the secret 👍

File: .github/workflows/stale.yml (hunk 1 above)
Comment (madkinsz) [reply .github/workflows/stale.yml:19]: They'd have 7 days to bump then another 30 days after that before it gets marked stale again 🤷‍♀️. I also lean generous on this though.

Comment (cicdw) [reply .github/workflows/stale.yml:19]: Gotcha gotcha - yea this is probably one of those things that will be easier to decide once we have a feel for it in practice.

Comment (bunchesofdonald) [reply .github/workflows/stale.yml:19]: I'm cool with 14, I'll bump that and merge this.

**Pull request merged**